test/Docfx.Build.RestApi.Tests/RestApiDocumentProcessorTest.cs

```diff
@@ -294,7 +294,7 @@ public void ProcessSwaggerWithInvalidLinksOverwriteShouldSucceedWithWarning()
             files.Add(DocumentType.Overwrite, new[] { "TestData/overwrite/rest.overwrite.invalid.links.second.md" });
             BuildDocument(files);
 
-            Assert.Equal(7, listener.Items.Count); // Additional warning for "There is no template processing document type(s): RestApi"
```

Comment:
Removed log is recorded as **Information** level.
So it seems no need to modify tests's log count assertions.